ts-apps/jl4-web/src/style.css

```diff
@@ -61,3 +61,16 @@
     border: 1px solid grey;
   }
 }
+
+.quick-input-widget {
+  position: absolute !important;
+  left: 25px !important;
+  right: auto !important;
+  top: 25px !important;
+}
+
+div[widgetid="editor.contrib.quickInputWidget"] {
```

Comment:
It's worth adding a quick comment to the effect that `editro...quickInputWidget` is an id internal to Monaco, if that is the case, as opposed to a public API.